File: plugins/kotlin/j2k/shared/tests/testData/newJ2k/switch/caseWithBlock.java
//method
void foo() {
  switch(a) {
    /* case 1 */ case 1: {
      int x = 1;
      System.out.println(x);
      break;
    }
    // case 2
    case 2: {
      int x = 2;
      System.out.println(x);
    }
    break;

    case 3: { // case 3 top
        System.out.println(3);
    } // case 3 bottom
  }
}

File: plugins/kotlin/j2k/shared/tests/testData/newJ2k/switch/caseWithBlock.kt
fun foo() {
    when (a) {
        /* case 1 */ 1 -> {
            val x = 1
            println(x)
        }
        // case 2
        2 -> {
            val x = 2
            println(x)
        }

        3 -> { // case 3 top
            println(3)
        } // case 3 bottom
    }
}
